### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by junitvn**
Redmine: https://edu-redmine.framgia.vn/issues/3674

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <activity android:name=".MainActivity" >
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
+
+</manifest>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: add end of file

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+</android.support.constraint.ConstraintLayout>
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: add end of file

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: remove hardcode

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
+</adaptive-icon>
```
Comment (DoanVanToan) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: add end of file

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
+</adaptive-icon>
```
Comment (DoanVanToan) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml:5]: add end of file

**Review by DoanVanToan — COMMENTED** [review]
- remove idea files

**Comment by DoanVanToan** [comment]
commented

**Comment by junitvn** [comment]
ready

**Comment by DoanVanToan** [comment]
commented
check comment cũ của anh nhé. 
Chú ý update spent time của redmine

**Comment by junitvn** [comment]
ready

**Comment by DoanVanToan** [comment]
commented
remove idea files

**Comment by junitvn** [comment]
ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**